cmd/otelcontribcol/builder-config.yaml

```diff
@@ -125,6 +125,7 @@ processors:
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/tailsamplingprocessor v0.109.0
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/transformprocessor v0.109.0
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/remotetapprocessor v0.109.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/googlecloudmonitoringreceiver v0.109.0
```

Comment:
Submitted a [fix](https://github.com/Internal-Cloudwerx/opentelemetry-collector-contrib/pull/1), the receiver is in the wrong group in the config